Tasks/CopyFilesOverSSHV0/sshhelper.ts

```diff
@@ -134,21 +136,19 @@ export class SshHelper {
      * @param path
      * @returns {Promise<boolean>}
      */
-    checkRemotePathExists(path: string) : Q.Promise<boolean> {
+    async checkRemotePathExists(path: string) : Promise<boolean> {
         var defer = Q.defer<boolean>();
 
         if(!this.sftpClient) {
             defer.reject(tl.loc('ConnectionNotSetup'));
         }
-        this.sftpClient.stat(path, function(err, attr) {
-            if(err) {
-                //path does not exist
-                defer.resolve(false);
-            } else {
-                //path exists
-                defer.resolve(true);
-            }
-        })
+        if (await this.sftpClient.stat(path)) {
```

Comment:
What a point to use `stat` method here? I assume we can use the [exist](https://www.npmjs.com/package/ssh2-sftp-client#sec-5-2-4) method from ssh2-sftp-client. So code be more clear:

```
const isPathExist: boolean = await this.sftpClient.exists(path);
defer.resolve(isPathExist);
```